lib/VM/JSLib/Intl.cpp

```diff
@@ -386,6 +387,33 @@ ExecutionStatus checkOptions(Runtime &runtime, const platform_intl::Options &) {
   return ExecutionStatus::RETURNED;
 }
 
+template <>
+ExecutionStatus checkOptions<platform_intl::DateTimeFormat>(
+    Runtime &runtime,
+    const platform_intl::Options &options) {
+  // https://www.ecma-international.org/wp-content/uploads/ECMA-402_9th_edition_june_2022.pdf
```

Comment:
The style in this file for quoting the spec is

```
// ECMA 402 11.1.2
35. Let hasExplicitFormatComponents be false.
36. For each row of table https://tc39.es/ecma402/#table-datetimeformat-components, except the header row, in table order, do
    e. If value is not undefined, then
        i. Set hasExplicitFormatComponents to true.
```

Perhaps add a 2023 to the first line, i.e.,

```
// ECMA 402 2023 11.1.2
```